src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String EMPTY_STRING = "";
+    private static final String VALID_LOGIN = "Alexandr";
+    private static final String INVALID_LOGIN = "Alex";
+    private static final String VALID_PASSWORD = "Alex12345";
+    private static final String INVALID_PASSWORD = "12345";
+    private static final int VALID_AGE = 20;
+    private static final int INVALID_AGE = 17;
+    private RegistrationService registrationService;
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setPassword(VALID_PASSWORD);
+        user.setLogin(VALID_LOGIN);
+        user.setAge(VALID_AGE);
+        registrationService = new RegistrationServiceImpl();
+    }

```

Comment:
you don't need any generic user for all tests - you should create users exclusively per test
```suggestion
    @BeforeEach
    void setUp() {
        registrationService = new RegistrationServiceImpl();
    }

```